src/Nethermind/Nethermind.TxPool/TxBroadcaster.cs

```diff
@@ -117,6 +124,13 @@ public void BroadcastOnce(ITxPoolPeer peer, Transaction[] txs)
 
         public void BroadcastPersistentTxs()
         {
+            if (_lastPersistedTxBroadcast + _minTimeBetweenPersistedTxBroadcast > DateTimeOffset.Now)
+            {
+                if (_logger.IsTrace) _logger.Trace($"Minimum time between persistent tx broadcast not reached.");
+                return;
+            }
+            _lastPersistedTxBroadcast = DateTimeOffset.Now;
+
```

Comment:
I would bring all of this under `if (_logger.IsTrace)` otherwise it is fine

Reply:
Are you sure? We could be spamming other peers during sync unintentially, they might disconnect us.

Reply:
Ah read it wrong